buildpack.md

```diff
@@ -1213,3 +1221,23 @@ name = "<dependency name>"
 [entries.metadata]
 version = "<dependency version>"
 ```
+
+### `0.8`
```

Comment:
I agree the headings are unclear. I think a unified reference for what has been deprecated when can be useful, to avoid having to scan through all the migration guides to find this information.

How about we get rid of the version headers, and add a clear, concise, consistent statement for each entry, like:

```
### Positional Arguments to `detect` and `build` Executables

_Deprecated in Buildpack API 0.8._
```

Reply:
Saw the 👍  so I've made this change.